### PR TITLE
Edit display name

### PR DESCRIPTION
**Pull request by ker-an**
A user can now update their display name by clicking the edit link and filling out a form. This is a workaround because Coral doesn't support the ability to update a display name under the My Profile tab.

image

### CODE REVIEW

File: origami.json [hunk 1]
```diff
@@ -18,7 +18,8 @@
 		"required": [
 			"Map",
 			"fetch",
-			"Array.from"
+			"Array.from",
+			"Element.prototype.remove"
```
Comment (JakeChampion) [inline origami.json]: 👌

**Comment by origamiserviceuser** [comment]
o-comments bundle size difference from 6.1.1 to 5343292254e041b663f3c20db56739d863020bae
js: 0.31kb increase (0.10kb/gzip)
An insignificant difference was also found for: css

File: origami.json (hunk 1 above)
Comment (ker-an) [reply origami.json]: FYI, the Page Kit team is uses the default Polyfill which `remove()` is apart of.

**Comment by JakeChampion** [comment]
I like the design of this :-)
The new comment design is much better than the previous one!

**Comment by origamiserviceuser** [comment]
o-comments bundle size difference from 7.0.0 to 06a44fb48cfd0f2dc4f684ddd308b1c304e4d0e2
js: 0.37kb increase (0.35kb/gzip)
css: 0.61kb decrease (-0.07kb/gzip)

**Comment by origamiserviceuser** [comment]
o-comments bundle size difference from 7.0.0 to 11f3d2685c084a9c1b62c92457f08278b289370f
js: 0.69kb increase (0.43kb/gzip)
css: 0.61kb decrease (-0.07kb/gzip)

**Pull request merged**